L1Trigger/L1TMuon/plugins/L1TMuonShowerProducer.cc

```diff
@@ -33,17 +33,17 @@ class L1TMuonShowerProducer : public edm::stream::EDProducer<> {
   edm::EDGetTokenT<l1t::RegionalMuonShowerBxCollection> showerInputToken_;
   int bxMin_;
   int bxMax_;
-  unsigned minNominalShowers_;
-  unsigned minTwoLooseShowers_;
+  bool enableOneNominalShower_;
+  bool enableTwoLooseShowers_;
 };
 
 L1TMuonShowerProducer::L1TMuonShowerProducer(const edm::ParameterSet& iConfig)
     : showerInputTag_(iConfig.getParameter<edm::InputTag>("showerInput")),
       showerInputToken_(consumes<l1t::RegionalMuonShowerBxCollection>(showerInputTag_)),
       bxMin_(iConfig.getParameter<int>("bxMin")),
       bxMax_(iConfig.getParameter<int>("bxMax")),
-      minNominalShowers_(iConfig.getParameter<unsigned>("minNominalShowers")),
-      minTwoLooseShowers_(iConfig.getParameter<unsigned>("minTwoLooseShowers")) {
+      enableOneNominalShower_(iConfig.getParameter<bool>("enableOneNominalShower")),
+      enableTwoLooseShowers_(iConfig.getParameter<bool>("enableTwoLooseShowers")) {
```

Comment:
Out of curiosity: What are these switches used for? At least in the uGMT I don't think it's likely we'll ever have only one of the bits enabled. If it's foreseen as as a feature let me know, I'll add it to my requirements.

Reply:
The minimal version we would like to commission for Run-3 is the "OneNominalShower" case. If you'd rather not have this switch at GMT, I could move it upstream to EMTF.

Reply:
That is where we intended to have the logic anyway.

Reply:
Yes, if that's easily possible I think that would be preferable. Also because we then can more easily accommodate "commissioning configurations" where maybe one sectors has more bits activated than the rest.

Reply:
Turns out I already have that option in the EMTF. So all I needed to do was remove some code in the uGMT.